cloud/tasks/logging/fields.go

```diff
@@ -16,6 +16,14 @@ type Field = log.Field
 ////////////////////////////////////////////////////////////////////////////////
 
 const ComponentYDB = "YDB"
+const ComponentS3 = "S3"
+const ComponentNbs = "NBS"
+const ComponentNfs = "NFS"
+const ComponentRunners = "RUNNERS"
+const ComponentScheduler = "SCHEDULER"
+const ComponentTask = "TASK"
+const ComponentApp = "APP"
+const ComponentClient = "CLIENT"
```

Comment:
Do we have to place "client", "nfs", "nbs" into "tasks" library?

Reply:
Lets add components only for tasks (for now)?

Reply:
I agree that it is better not to place these components  into "tasks" library. Removed them, now there are no changes outside "tasks" in this pr.

Is it ok to add "client", "nfs", "nbs", "app" components in separate pr?